installer/PowerToysSetup/Product.wxs

```diff
@@ -127,7 +128,7 @@
             <Custom Action="TelemetryLogUninstallSuccess" After="InstallFinalize">
                 Installed and (NOT UPGRADINGPRODUCTCODE) AND (REMOVE="ALL")
             </Custom>
-            <Custom Action="UnApplyModulesRegistryChangeSets" Before="InstallFinalize">
+            <Custom Action="UnApplyModulesRegistryChangeSets" Before="RemoveFiles">
```

Comment:
Why did this change?

Reply:
I think because we need some of the files we then remove in the next step.

Reply:
@htcfreek is correct, to unapply the monaco handler registry changes from CA installer, we need to read languages.json file to get a list of extensions the handler is used for.